scripts/check_site.lua

```diff
@@ -1,4 +1,3 @@
-local cjson = require 'cjson'
```

Comment:
I'm currently evaluating whether we can switch to `lua-jsonc` unconditionally, removing the need to support two different loaders in `check_site.lua`.

Reply:
https://github.com/freifunk-gluon/gluon/pull/2234